src/components/Rubric.js

```diff
@@ -1,177 +1,171 @@
 import React, { Component } from 'react';
+import Criteria from './Criteria';
 import * as Actions from '../actions/assessment';
 import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
 import { withRouter } from 'react-router'
 import uuidv1 from 'uuid/v1';
 
-const CompetencyButton = props => {
+const CompetencyButton = withRouter(props => {
   return (
     <li className={props.isActive}
       onClick={() => {
         props.setActiveComp(props.index)
       }}>
-      <a href="#dashboard-2" role="tab" data-toggle="tab" aria-expanded="true">
+      <a href="" onClick={(e) => e.preventDefault()} role="tab" data-toggle="tab" aria-expanded="true">
         <i className="material-icons">{props.icon}</i> {props.name}
-      </a>
-    </li>
-  )
-}
-
-const Criteria = props => {
-  return (
-    <tr>
-      <td>
-        <div className="checkbox">
-          <label>
-            <input type="checkbox" name="optionsCheckboxes" defaultChecked={false} />
-          </label>
-        </div>
-      </td>
-      <td>{props.text}</td>
-    </tr>
-  )
-}
-
-class Rubric extends Component {
-
-  constructor(props) {
-    super(props);
-    this.state = {
-      activeCompetencyIndex: 0
+        </a>
+      </li>
+    )
+  })
+
+  class Rubric extends Component {
+
+    constructor(props) {
+      super(props);
+      this.state = {
+        activeCompetencyIndex: 0
+      }
+      this.icons = [ "dashboard", "explore", "code", "backup", "lock", "bug_report", "line_style", "perm_identity", "star_rate" ]
+      this.renderCompetencies = this.renderCompetencies.bind(this);
+      this.renderCompetencyButtons = this.renderCompetencyButtons.bind(this);
+      this.renderCriteriaForLevel = this.renderCriteriaForLevel.bind(this);
+      this.renderLevels = this.renderLevels.bind(this);
+      this.getIsFetching = this.getIsFetching.bind(this);
```

Comment:
Do you need to bind like this?

You can also just use fat arrow functions when you declare the function like this on line 124:

`getIsFetching = () => {`

This will bind the this to the function and you have to have the extra line at the top of the file..